### ISSUE TITLE
Enhancement: Automatically closing the workitem detail panel

### ISSUE THREAD

**Issue by ldimaggi** [fabric8io/fabric8-planner#1197]
If a user is working on/viewing a workitem's details and clicks outside of the open detail panel, for example, to "quick create" a new workitem, the detail panel remains open. It would be useful if it could be closed automatically.

![screenshot from 2017-03-15 08-17-25](https://cloud.githubusercontent.com/assets/642621/23954404/94bf8aee-096d-11e7-9a0b-c407a2e783e2.png)

**Comment by Mgranfie** [comment]
Agreed, Len, there would just have to be warning, as defined in that
InVision doc... located here:

https://fabric8io.github.io/fabric8-ux/invision-links.html

**Comment by ldimaggi** [comment]
See: https://openshift.io/openshiftio/openshiftio/plan/detail/1039
